.storybook/mobileWrapper.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import GlobalStyles from '../src/GlobalStyles'
+
+export default (renderComponent) => (
+  <>
+    <GlobalStyles />
+    <div style={{ padding: 20, width: 400 }}>{renderComponent()}</div>
+  </>
+)
```

Comment:
Vielleicht könntest du die Datei umbenennen in preview.js, damit sie einen allgemeineren Namen hat. Eventuell willst du für Storybook später ja auch noch den Memory Router einfügen, dann ist das praktisch :)

Reply:
Das ist eine gute Idee!